### ISSUE TITLE
User tokens

### ISSUE THREAD

**Issue by nunomoreno** [nextcloud/zenodo#5]
Hi,

currently it is only possible to link a single token. Would it be possible to have a token per user?

**Comment by daita** [comment]
I don't see the point, would you mind be more specific on the usecase ?

**Comment by daita** [comment]
But a single token is enough for your users to upload their documents

**Comment by nunomoreno** [comment]
Correct! But always to the same zenodo account... unless I'm doing something wrong. Suppose that I want to use zenodo as cold storage: if I select to have it private, I cannot go to my zenodo account and see it as it was uploaded using some else token.

**Comment by theWeird** [comment]
I would also prefer to have settings for zenodo API key in each users profile.  
Maybe open a different dialogue, if a user wants to post something to zenodo but hasn't defined an api key yet.
